Komet/ZGCommitTextView.m

```diff
@@ -44,13 +45,6 @@ - (void)setContinuousSpellCheckingEnabled:(BOOL)continuousSpellCheckingEnabled
 	[super setContinuousSpellCheckingEnabled:continuousSpellCheckingEnabled];
 }
 
```

Comment:
There's no longer any need to override this method, as managing the defaults state for spelling correction is now done by `ZGUserDefaults`.